### ISSUE TITLE
Question: How do I create different success.json based on a param?

### ISSUE THREAD

**Issue by JonWallsten** [smollweide/node-mock-server#7]
I have not yet figured this one out and need help. 

Folder structure:
#test#{id}
#test#{id}/GET
#test#{id}/GET/mock

Example:
**Call: /rest/test/1**
Response should be:
```
{
      "id": 1,
      name: "John"
}
``` 
**Call: /rest/test/2**
Response should be:
```
{
      "id": 2,
      name: "Sven"
}
``` 

Edit. Also i don't figure out how to show error.json if no ID is supplied when doing the GET.

**Comment by smollweide** [comment]
@JonWallsten Currently it's just possible to use query params as ejs variables [query param in ejs](https://github.com/smollweide/node-mock-server/blob/master/example_rest_folder/products/%23%7BproductCode%7D/GET/mock/request-data.json)

I will add the dynamic path to ejs object.
The solution (with future version 0.7.2 [Issue 9](https://github.com/smollweide/node-mock-server/issues/9)) will be:
```
{
      "id": <%=params.id%>,
      name: "<%-faker.name.firstName();%>"
}
```

You question about how to change the mock response:
- Change the expected response by using the UI
- Use the query param `_expected=error`
- Use the request header `_expected=error`

[Expected response](https://github.com/smollweide/node-mock-server#expected-response)

An automatic check for not existing dynamic path is currently not implemented.
[Issue 8](https://github.com/smollweide/node-mock-server/issues/8)

**Comment by JonWallsten** [comment]
Cool! Then I'll wait for the next release.

Ah, i see! I thought it could be done automagically! ;)
Thanks!

**Comment by JonWallsten** [comment]
And as a feature request it would be cool if you could use different success files for different params.
Like success{id}.json and the you name the files success1.json, success2.json, and so on!

**Comment by JonWallsten** [comment]
```
{
	"id": 1,
	"campaignId": <%=id%>,
	"name": "Test"
}
```

I get a reference error: 
"id" is not defined. 
path: undefined.
Is it because I'm using /rest/test/1 instead of /rest/test/?id=1 ?

**Comment by smollweide** [comment]
@JonWallsten 
Yes that's because you use /rest/test/1 instead of /rest/test/?id=1
`0.7.2` is released - `npm install node-mock-server@0.7.2`
which includes the two issues, has arisen from your questions.
- [milestone 0.7.2](https://github.com/smollweide/node-mock-server/issues?q=milestone%3A0.7.2+is%3Aclosed)
- [issue8](https://github.com/smollweide/node-mock-server/issues/8)
- [issue9](https://github.com/smollweide/node-mock-server/issues/9)

With the new release you can use this:
```
{
      "id": <%=params.id%>,
      name: "<%-faker.name.firstName();%>"
}
```

I hope it works as you had assumed.
Thanks for your feedback and questions.

**Comment by JonWallsten** [comment]
Thank you! That's awesome!
